benchmark/src/walltime.cc

```diff
@@ -65,9 +65,15 @@ class WallTimeImp
   static WallTimeImp& GetWallTimeImp() {
     static WallTimeImp imp;
 #if __cplusplus >= 201103L
-    static_assert(std::is_trivially_destructible<WallTimeImp>::value,
-                  "WallTimeImp must be trivially destructible to prevent "
-                  "issues with static destruction");
+	#if defined(__GNUC__) && (__GNUC__*100 + __GNUC__MINOR__ < 408)
```

Comment:
That was intentional for readability as it's a nested if. Do you prefer this to not be indented?